css/site.styl

```diff
@@ -431,7 +431,8 @@ img
   height: 100%
 
 #archive-editorial-titles
-  max-width: 500px
+// >>> be nice if this value had a relation to container or something, or to other content like this. It could also be an indent not width centered
+  max-width: 350px
```

Comment:
i think this should have `margin-left: $margin-mid` instead of max-width and centered